department-of-veterans-affairs/abd-vro
# Create new workflow for only deploying db-init

## What was the problem?

With removal of VRO App upcoming, we can no longer rely on that deployment workflow to deploy db-init alongside vro-app

Associated tickets or Slack threads:
- #2697 

## How does this fix it?[^1]

Adds new workflow and helm charts for future testing before removing existing functionality around vro app completely 

## How to test this PR
- Will use the new workflow to test the helm chart 

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
